styles/palette.css

```
@@ -20,19 +20,42 @@
   --stanford-10-black-rgb: 234, 234, 234;
 
   /* https://identity.stanford.edu/design-elements/color/accent-colors/ */
+  --stanford-palo-alto-rgb: 23, 94, 84;
+  --stanford-palo-alto-light-rgb: 45, 113, 111;
+  --stanford-palo-alto-dark-rgb: 1, 66, 64;
+  --stanford-palo-verde-rgb: 39, 153, 137;
+  --stanford-palo-verde-light-rgb: 89, 179, 169;
+  --stanford-palo-verde-dark-rgb: 1, 126, 124;
+  --stanford-palo-verde-light: var(rgb(--stanford-palo-verde-light-rgb));
```

```suggestion
  --stanford-palo-verde-light: rgb(var(--stanford-palo-verde-light-rgb));
```

Should all these `var(rgb` be `rgb(var`?

---

nice catch